### DISCUSSION TITLE
Core Plot master does not compile

### DISCUSSION THREAD

**Discussion by o-bhave** [core-plot/core-plot#481] [General]
I am getting several compilation errors when I try to build latest from master for macOS

core-plot/framework/Source/_CPTAnimationPlotRangePeriod.m:11:31 Cast from 'IMP' (aka 'void (*)(void)') to 'GetterType' (aka 'NSValue *(*)(__strong id, SEL)') converts to incompatible function type

core-plot/framework/Source/_CPTXYTheme.m:41:21 Mixing declarations and code is incompatible with standards before C99

core-plot/framework/Source/_CPTXYTheme.m:41:21 Mixing declarations and code is incompatible with standards before C99

Can you please address these. I had raised similar issue several months ago and I was told that it is fixed. But I still see these errors.

Omkar Bhave

**Comment by eskroch** [comment]
Use the Core Plot `release-2.4` branch. It will work with any current Xcode release.
